wire-ios-sync-engine/Tests/Source/Calling/CallKitManagerTests.swift

```diff
@@ -500,22 +500,22 @@ class CallKitManagerTest: DatabaseTest {
     }
 
     /* Disabled for now, pending furter investigation
-    func testThatCallAnswerActionFailWhenCallCantBeJoined() {
-        // given
-        let otherUser = self.otherUser(moc: self.uiMOC)
-        let provider = MockProvider(foo: true)
-        let conversation = self.conversation(type: .oneOnOne)
-
-        sut.reportIncomingCall(from: otherUser, in: conversation, video: false)
-        let action = MockCallAnswerAction(call: sut.callUUID(for: conversation)!)
-        self.sut.provider(provider, perform: action)
-
-        // when
-        mockWireCallCenterV3.update(callState: .terminating(reason: .lostMedia), conversationId: conversation.remoteIdentifier!, callerId: otherUser.remoteIdentifier, isVideo: false)
-        XCTAssertTrue(waitForAllGroupsToBeEmpty(withTimeout: 0.5))
-        // then
-        XCTAssertTrue(action.hasFailed)
-    }
+     func testThatCallAnswerActionFailWhenCallCantBeJoined() {
```

Comment:
just passing through this, is there a ticket somewhere about this? if so put a comment with the Jira ticket, else create one

Reply:
or is it still relevant? to the current code

Reply:
I have no idea, it only made it into this PR because the indentation changed somehow